dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/action/ViewCMSMaintenanceAction.java

```diff
@@ -63,17 +38,7 @@
 import com.dotmarketing.util.UtilMethods;
 import com.dotmarketing.util.WebKeys;
 import com.dotmarketing.util.ZipUtil;
-import com.google.common.collect.ImmutableList;
-import com.liferay.portal.ejb.ImageLocalManagerUtil;
-import com.liferay.portal.ejb.PortletPreferencesLocalManagerUtil;
-import com.liferay.portal.model.Company;
-import com.liferay.portal.model.User;
-import com.liferay.portal.util.PortalUtil;
-import com.liferay.portlet.ActionRequestImpl;
-import com.liferay.portlet.ActionResponseImpl;
-import com.liferay.util.FileUtil;
-import com.liferay.util.servlet.SessionMessages;
-import com.liferay.util.servlet.UploadPortletRequest;
+
 import java.io.BufferedOutputStream;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.BufferedOutputStream'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)